radio/util/font2png.py

```diff
@@ -91,6 +91,12 @@ def getFontTopBottom():
             pass
         elif c == "." and fontsize <= 8:
             painter.drawPoint(width, fontsize)
+        elif c == u"↑" or c == u"↓":
+            rect = metrics.boundingRect(c)
+            if fontsize >= 24:
+                painter.drawText(width - rect.left() + 1, fontsize - 2, c)  # fontsize-bottom+1 -17 / 7
+            else:
+                painter.drawText(width - 1, fontsize, c)  # fontsize-bottom+1 -17 / 7            
```

Comment:
Ignore my advice, I looked only at the one changeset. Your final solution is OK.